client/src/App.js

```diff
@@ -1,25 +1,20 @@
 import React from 'react';
-import logo from './logo.svg';
+import{Route,Switch} from 'react-router-dom';
 import './App.css';
```

Comment:
is APP.css still used ? if not you can remove this import

Reply:
Its not in use
will remove it.